src/definition/livechat/IPostLivechatRoomClosed.ts

```diff
@@ -12,7 +12,10 @@ export interface IPostLivechatRoomClosed {
      * @param livechatRoom The livechat room which is closed.
      * @param read An accessor to the environment
      * @param http An accessor to the outside world
-     * @param persistence An accessor to the App's persistence
+     * @param persis An accessor to the App's persistence
+     * @param modify An accessor to the modifier
      */
-    [AppMethod.EXECUTE_POST_LIVECHAT_ROOM_CLOSED](room: ILivechatRoom, read: IRead, http: IHttp, persistence: IPersistence): Promise<void>;
+    [AppMethod.EXECUTE_POST_LIVECHAT_ROOM_CLOSED](
+        room: ILivechatRoom, read?: IRead, http?: IHttp, persis?: IPersistence, modify?: IModify,
```

Comment:
```suggestion
        room: ILivechatRoom, read: IRead, http: IHttp, persis: IPersistence, modify?: IModify,
```

There is no need to mark the existing parameters as optional, only the new one